File: 2344/2344.py
notas = int(input())

if(notas == 0):
    print('E')
elif(notas <= 1 or notas <=35):
    print('D') 
elif(notas <= 36 or notas <=60):
    print("C")
elif(notas <=61 or notas <=85):
    print("B")
elif(notas <=86 or notas < 101):
    print("A")               

File: 2344/2344.c
#include<stdio.h>

void notas( int notas){

    if(notas == 0){
    printf("E\n");
    }
else if(notas <= 1 || notas <=35){
    printf("D\n");
     }
else if(notas <= 36 || notas <=60){
    printf("C\n");
    }
else if(notas <=61 || notas <=85){
    printf("B\n");
    }
else if(notas <=86 || notas < 101){
    printf("A\n");
     }
}

int main(){
    int valor;
    scanf("%d",&valor);
    notas(valor);
    return 0;
}